pom.xml

```diff
@@ -94,6 +94,198 @@
     <commons-lang.version>2.6</commons-lang.version>
     <openlineage.version>1.22.0</openlineage.version>
   </properties>
+  <dependencies>
```

Comment:
Should some (all?) of these dependencies be on the `-core` package? Things like `google-cloud-bigtable` and such are dependencies for core, but since the parent package doesn't really have any source code we can leave these out of here and keep this pom focused on package assembly

Reply:
core is just to provide all the source file. Created a new module (spark-bigtable). scala2.12 and scala2.13 are placed inside it. common dependencies are part of this new module now.